### PR TITLE
new_css

### PR DESCRIPTION
**Pull request by AuroraChris**

### CODE REVIEW

File: src/sass/default.scss [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
-table
+.BookList
 {
   border-collapse: collapse;
 }
 
-table,
+.BookList,
 th,
```
Comment (byronharvey24) [inline src/sass/default.scss]: probably want to give classes to th and td as well!

File: src/sass/default.scss [hunk 2]
```diff
@@ -24,32 +24,32 @@ td
   border: none;
 }
 
-h1
+.BookList-header
```
Comment (byronharvey24) [inline src/sass/default.scss]: These can be nested within booklist!

.BookList {
 border: none

&-header {
margin-bottom: 0;
}
}

File: src/sass/default.scss (hunk 1 above)
Comment (AuroraChris) [reply src/sass/default.scss]: Got it, will do now

File: src/sass/default.scss (hunk 2 above)
Comment (AuroraChris) [reply src/sass/default.scss]: Got it, will fix now

**Pull request merged**